child_compassion/wizards/child_search_attributes.py

```diff
@@ -0,0 +1,124 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2015 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Emanuel Cino
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+from openerp import models, api, fields, _
+
+
+class ImportReview(models.TransientModel):
+
+    _name = 'child.search.attributes'
+
+    ##########################################################################
+    #                                 FIELDS                                 #
+    ##########################################################################
+
+    name = fields.Char()
+    type = fields.Selection(selection=[
+        ('Available', _('Available')),
+        ('Change Commitment Hold', _('Change Commitment Hold')),
+        ('Consignment Hold', _('Consignment Hold')),
+        ('Delinquent Mass Cancel Hold', _('Delinquent Mass Cancel Hold')),
+        ('E-Commerce Hold', _('E-Commerce Hold')),
+        ('Inactive', _('Inactive')),
+        ('Ineligible', _('Ineligible')),
+        ('No Money Hold', _('No Money Hold')),
+        ('Reinstatement Hold', _('Reinstatement Hold')),
+        ('Reservation Hold', _('Reservation Hold')),
+        ('Sponsor Cancel Hold', _('Sponsor Cancel Hold')),
+        ('Sponsored', _('Sponsored')),
+        ('Sub Child Hold', _('Sub Child Hold'))])
+    hold_expiration_date = fields.Datetime()
+    primary_owner = fields.Char()
+    secondary_owner = fields.Char()
+    no_money_yield_rate = fields.Float()
+    yield_rate = fields.Float()
+
+    ##########################################################################
+    #                             FIELDS METHODS                             #
+    ##########################################################################
+
+    @api.multi
+    def send(self):
+
+        holds = self.env['compassion.hold']
+        messages = self.env['gmc.message.pool']
+        global_child = self.env['compassion.childpool.search'].browse(
+            self.env.context.get('active_id'))
+
+        for child in global_child.global_child_ids:
+            # Save children form global children to compassion children
+
+            child_vals = {
+                'global_id': child.global_id,
```

Comment:
instead of copying all values, you could use method `read` of an object to get that dictionary. See https://www.odoo.com/documentation/8.0/reference/orm.html#openerp.models.Model.read